A patch to tensorflow internals BUILD file to export
NEON_2_SSE.h as part of the //tensorflow/lite/kernels/internal:install_nnapi_extra_headers
build target, on Intel platforms only.

diff --git a/tensorflow/lite/kernels/internal/BUILD.orig b/tensorflow/lite/kernels/internal/BUILD
index 5bafcdc..dc959c5 100644
--- a/tensorflow/lite/kernels/internal/BUILD.orig
+++ b/tensorflow/lite/kernels/internal/BUILD
@@ -859,7 +859,17 @@ transitive_hdrs(
     deps = [
         "//third_party/eigen3",
         "@gemmlowp",
-    ],
+    ] + select({
+        ":haswell": tflite_deps_intel,
+        ":ios_x86_64": tflite_deps_intel,
+        ":k8": tflite_deps_intel,
+        ":x86": tflite_deps_intel,
+        ":x86_64": tflite_deps_intel,
+        ":darwin": tflite_deps_intel,
+        ":darwin_x86_64": tflite_deps_intel,
+        ":freebsd": tflite_deps_intel,
+        "//conditions:default": [],
+    }),
 )

 # ---------------------------------------------------------
